src/index.html

```diff
@@ -1,13 +1,285 @@
+
+          <p class="place__address">
+            <a href="#place1">Marina Palms</a>
+            / North Miami Beach, FL 33139
+          </p>
+        </div>
+
+        <div class="place">
+          <div class="place__photos">
+            <img class="place__photo place__photo--5" src="./images/photos/5.jpg" alt="Place 6">
+            <img class="place__photo place__photo--6" src="./images/photos/6.jpg" alt="Place 5">
+          </div>
+
+          <p class="place__address">
+            <a href="#place1">Marina Palms</a>
+            / North Miami Beach, FL 33140
+          </p>
+        </div>
+
+        <div class="gallery__button-container">
+          <a
+            class="gallery__button button button--secondary"
+            href="#view-all"
+          >
+            View all
+          </a>
+        </div>
+      </div>
+    </section>
+
+    <section class="page__section testimonials" id="testimonials">
+      <div class="container">
+        <h2 class="section-title">Testimonials</h2>
+
+        <div class="testimonials__content">
+          <div class="testimonials__item-1 testimonial">
+            <img
+              src="./images/people/1.jpg"
+              alt="Name1 Lastname1"
+              class="testimonial__photo"
+            >
+
+            <p class="testimonial__name">Leslie Mckinney</p>
+            <p class="testimonial__title">CEO</p>
+
+            <p class="testimonial__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+
+          <div class="testimonials__item-2 testimonial">
+            <img
+              src="./images/people/2.jpg"
+              alt="Name2 Lastname2"
+              class="testimonial__photo"
+            >
+
+            <p class="testimonial__name">Leslie Mckinney</p>
+            <p class="testimonial__title">CEO</p>
+
+            <p class="testimonial__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+
+          <div class="testimonials__item-3 testimonial">
+            <img
+              src="./images/people/3.jpg"
+              alt="Name3 Lastname3"
+              class="testimonial__photo"
+            >
+
+            <p class="testimonial__name">Leslie Mckinney</p>
+            <p class="testimonial__title">CEO</p>
+
+            <p class="testimonial__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="page__section contact-us" id="contact-us">
+      <div class="container">
+        <h2 class="section-title">Contact us</h2>
+
+        <div class="contact-us">
+          <div class="grid">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <h3 class="contact-us__info-title">Phone</h3>
+                +1 234 5555-55-55
```

Comment:
mb it's a link?

Reply:
It looks like a text but can make it a link